packages/blocks/src/__internal__/utils/gesture.ts

```diff
@@ -236,16 +238,26 @@ export function initMouseEventHandlers(
     );
   };
 
+  const selectionChangeHandler = debounce(e => {
+    if (isDragging) {
+      return;
+    }
+
+    onSelectionChange(e as Event);
+  }, 100);
+
   container.addEventListener('mousedown', mouseDownHandler);
   container.addEventListener('mousemove', mouseMoveHandler);
   container.addEventListener('contextmenu', contextMenuHandler);
   container.addEventListener('dblclick', dblClickHandler);
+  document.addEventListener('selectionchange', selectionChangeHandler);
```

Comment:
This looks a bit overkill, since the input elements outside of BlockSuite may be unexpectedly handled. Why not add this listener to the `container`?

Reply:
Because event selectionchange only work on document.🥲